torch/jit/__init__.py

```diff
@@ -1568,7 +1595,13 @@ def __init__(self, original, stubs):
                 elif isinstance(item, (Parameter, Module, Attribute)):
                     if isinstance(item, (ModuleList, Sequential)):
                         # These are in __constants__, so ignore them here
-                        continue
+
+                        if not torch._C._jit_recursive_script():
+                            # For recursive script, these are constantified after
```

Comment:
Maybe add test for an example of when these are constantified and then attempted to be modified, if that applies